include/vrv/options.h

```diff
@@ -94,8 +94,10 @@ class Option {
     virtual bool SetValueDbl(double value);
     virtual bool SetValueArray(const std::vector<std::string> &values);
     virtual bool SetValue(const std::string &value);
-    virtual std::string GetStrValue() const;
-    virtual std::string GetDefaultStrValue() const;
+    virtual std::string GetStrValue() const = 0;
```

Comment:
Maybe this is just me not understanding a C++ism, but why do you set a string to 0?

Reply:
In C++ that is how you define a _pure virtual function_ which must be overridden in derived classes. This makes Option an abstract class. It was somewhat required before by putting a runtime check `assert(false)` in the implementation, but it is much safer and standard to enforce this at compile time.

Reply:
Yes, there are a few places where we should add this.